lib/Doctrine/ORM/AbstractQuery.php

```diff
@@ -979,8 +980,8 @@ public function iterate($parameters = null, $hydrationMode = null)
      * Executes the query and returns an iterable that can be used to incrementally
      * iterate over the result.
      *
-     * @param ArrayCollection|mixed[] $parameters    The query parameters.
-     * @param string|int|null         $hydrationMode The hydration mode to use.
+     * @param ArrayCollection|array|mixed[] $parameters    The query parameters.
```

Comment:
|array seems useless given mixed[]

Reply:
@orklah see https://github.com/doctrine/orm/issues/8624 for why this is actually needed

Reply:
yeah, I was just reading that, thanks for the info :)